src/components/app.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
+import Profile from '../containers/profile/profile.jsx';
```

Comment:
`import Profile from '../containers/profile/profile.jsx';`

Instead, use the template

`import Profile from '../containers/profile';`

And rename `profile.jsx` into `index.jsx`